catalog/ui/search-ui/search-redirect/pom.xml

```diff
@@ -64,9 +58,6 @@
                         <!--<Web-ContextPath>/search</Web-ContextPath>-->
                         <!--<_wab>src/main/webapp</_wab>-->
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
```

Comment:
❓ If you don't embed jetty-servlets, wouldn't you need to import the packages exported by the jetty-servlets bundle?